### ISSUE TITLE
IPA: baby gamma shape is deprecated

### ISSUE THREAD

**Issue by moyogo** [Kelvinsong/swiftday#5]
The glyph of ɤ U+0264 currently has the baby gamma shape. That shape was deprecated by the IPA in 1989 during the Kiel Convention. The ram’s horn shape (with the terminals curving down) is preferred.

**Comment by tayloraswift** [comment]
I changed the shape better now? https://github.com/Kelvinsong/swiftday/commit/f1bdffd5966f5c2fe86a3ff56d0f9c60f393da87

**Comment by moyogo** [comment]
Yes that's better but the right side could point down more, a tiny bit or a lot more.
For a lot more see Doulos SIL or Charis SIL
![capture d ecran 2015-05-21 a 10 25 23](https://cloud.githubusercontent.com/assets/1923747/7745531/c0da4a36-ffa3-11e4-8188-2c03664db5ca.png)

I'd say the italic should be quite different, the right arm should finish curving down but I guess there's room for interpretation given the style.

Doulos SIL qnd Charis SIL also have it a bit wider.

Here's the snippet from the “Report on the 1989 Kiel Convention”, _Journal of the International Phonetic Association_, volume 19, issue 12, December 1989
![capture d ecran 2015-05-21 a 10 30 33](https://cloud.githubusercontent.com/assets/1923747/7745624/a4cdc510-ffa4-11e4-821f-325dcd997c35.png)
